MerlinAU.sh

```diff
@@ -5422,7 +5443,11 @@ _SelectOfflineUpdateFile_()
     done
 
     # Extract or prompt for firmware version #
-    _GetOfflineFirmwareVersion_ "$selected_file"
+    if ! _GetOfflineFirmwareVersion_ "$selected_file"
+    then
+        printf "Operation was cancelled by user. Exiting.\n"
+        return 1
```

Comment:
Allow the user to exit this screen without making a selection. (Return to advanced menu screen)